whatshap/cli/stats.py

```diff
@@ -64,6 +65,17 @@ def variants(self):
     def count_snvs(self):
         return sum(int(variant.is_snv()) for variant in self.phases)
 
+    def split(self, split_left: int, split_right: int) -> Tuple["PhasedBlock", "PhasedBlock"]:
+        """Split a phaseblock in two based on given positions"""
```

Comment:
* Should be "split *this* phase block".
* The docstring isn’t detailed enough ("based on" is unclear). Please describe `split_left` and `split_right` (because intuitively I would expect a `split()` function to only have one split position argument).
* Should there be an `assert split_left <= split_right`?

Reply:
Agreed! I will update the docs here and a the `assert split_left <= split_right` is probably a good call.